CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Treasure Agent 4 changelog
 
+## Release v4.4.0 - 2022/07/29
+
+### News
+
+* Update fluentd to 1.15.1
+* Drop Ubuntu 16.04 "Xenial Xerus" support
+
+### Core component
+
+* ruby v2.7.6
+* jemalloc v3.6.0
+* fluentd v1.15.1 (update)
+
+### Core gems
+
+* bundler v2.3.18 (update)
+* msgpack v1.5.3 (update)
+* cool.io v1.7.1
+* serverengine v2.3.0 (update)
+* oj v3.13.17 (update)
+* async-http v0.56.6 (update)
+* http_parser.rb v0.8.0
+* yajl-ruby v1.4.3
+* sigdump v0.2.4
+* tzinfo v2.0.5 (update)
+* tzinfo-data v1.2022.1
+
+### Bundled plugins and gems
```

Comment:
Shouldn't we add the following gems?
They seem to be missing since [they are newly added at v4.3.1](
https://github.com/fluent/fluent-package-builder/pull/378/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR8-R12).

* fluent-plugin-opensearch v1.0.7 (update)
* fluent-plugin-metrics-cmetrics v0.1.2 
* fluent-plugin-calyptia-monitoring v0.1.3
* fluent-plugin-windows-exporter v1.0.0 (to `On Windows`)

Reply:
Thanks, fixed in [2e49d1e](https://github.com/fluent/fluent-package-builder/pull/405/commits/2e49d1e31bf69a5a7886b182f6705e09517637ab)